forge-gui/res/cardsfolder/a/adrestia.txt

```diff
@@ -6,11 +6,7 @@ K:Landwalk:Island
 T:Mode$ Attacks | ValidCard$ Card.Self | CheckSVar$ X | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ Whenever CARDNAME attacks, if an Assassin crewed it this turn, draw a card. CARDNAME becomes an Assassin in addition to its other types until end of turn.
 SVar:TrigDraw:DB$ Draw | NumCards$ 1 | SubAbility$ DBAnimate
 SVar:DBAnimate:DB$ Animate | Defined$ Self | Types$ Assassin
-T:Mode$ BecomesCrewed | ValidVehicle$ Card.Self | Execute$ RememberCrew | Static$ True
-SVar:RememberCrew:DB$ PumpAll | ValidCards$ Assassin.TriggeredCrew | RememberPumped$ True
-T:Mode$ Phase | Phase$ End of Turn | Static$ True | Execute$ DBCleanup
-SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
-SVar:X:Remembered$Amount
+SVar:X:Crewed$Valid Assassin
```

Comment:
this isn't fully accurate because the condition should look back in time
But I don't think crewedByThisTurn can be fully changed to the LKI paidList because some others use it for ingame stuff?

Reply:
i changed the crewedThisTurn into an LKI list

feel free to test that

Reply:
ok works fine